extension/src/experiments/columns/collect/util.ts

```diff
@@ -15,7 +15,7 @@ export const limitAncestorDepth = (
     concatenated layer itself counts as one; because of this, we must subtract 3
     from what we want the final layer count to be.
   */
-  const convertedLimit = limit - 3
+  const convertedLimit = (limit >= 3 ? limit : 3) - 3
```

Comment:
Currently, this function only works correctly if the limit is at least 3. There's probably a way to rewrite the logic to allow for 1 or 2 layers though. Should this be done in a followup?